app/frontend/Chat/MessageView/Message.svelte

```diff
@@ -74,6 +74,21 @@
   $: reactions = $message.reactions;
 
   $: headHTML = `<style>\n${cssBody}\n${cssContent}\n</style>`;
+
+  let messageE: HTMLDivElement;
+  let maxWidth: number;
+
+  onMount(() => {
+    observeMaxWidth();
+  });
+
+  function observeMaxWidth() {
+    const observer = new ResizeObserver((entries) => {
+      const el = entries[0];
+      maxWidth = el.contentRect.width * 0.75;
+    });
+    observer.observe(messageE.parentElement);
```

Comment:
Can you move this into the WebView, please? The WebView can access its `parentElement` (or grandparent) just the same, so there should be no need to put it outside the component.

Reply:
Ok, will do.

Reply:
Thanks